paragraph/scheduling/logger_test.cc

```diff
@@ -0,0 +1,170 @@
+/* Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "paragraph/scheduling/logger.h"
+
+#include <filesystem>
+#include <fstream>
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "absl/status/status.h"
+#include "gtest/gtest.h"
+#include "paragraph/graph/graph.h"
+#include "paragraph/scheduling/graph_scheduler.h"
+#include "paragraph/shim/test_macros.h"
+
+std::string get_testfile_name(const std::string& _basename) {
```

Comment:
Google doesn't like preceding underscores, but Nic does :)

Reply:
fixed here and in graph_test.cc